src/student_success_tool/assets/pdp/features_table.toml

```diff
@@ -8,20 +8,20 @@ num_credits_earned = { name = "number of credits earned this term" }
 course_id_nunique = { name = "number of unique courses taken this term" }
 course_subject_nunique = { name = "number of unique course subjects taken this term" }
 course_subject_area_nunique = { name = "number of unique course subject areas taken this term" }
-course_level_mean = { name = "avg. course level this term" }
-course_level_std = { name = "std. dev. in course levels this term" }
-course_grade_numeric_mean = { name = "avg. numeric course grade this term" }
-course_grade_numeric_std = { name = "std. dev. in numeric course grades this term" }
-section_num_students_enrolled_mean = { name = "avg. number of students enrolled per section this term" }
-section_num_students_enrolled_std = { name = "std. dev. in number of students enrolled per section this term" }
+course_level_mean = { name = "average course level this term" }
+course_level_std = { name = "standard deviation in course levels this term" }
+course_grade_numeric_mean = { name = "average numeric course grade this term" }
+course_grade_numeric_std = { name = "standard deviation in numeric course grades this term" }
+section_num_students_enrolled_mean = { name = "average number of students enrolled per section this term" }
+section_num_students_enrolled_std = { name = "standard deviation in number of students enrolled per section this term" }
 num_courses_core_course_y = { name = "number of core courses taken this term" }
 num_courses_enrolled_at_other_institution_s_y = { name = "number of courses taken this term while enrolled at another institution" }
 'num_courses_course_subject_area_(\d+)' = { name = "number of courses taken in subject area {} this term" }
-'num_courses_course_id_(.*)' = { name = "number of times course '{}' taken this term" }
+'num_courses_course_id_(.*)' = { name = "course '{}' taken this term" }
```

Comment:
technically it is in fact the number of times the course is taken. is every value == 1?

Reply:
in the vast majority of cases, the "per term" count for a course will likely either be 1 or 0. This feature specifically confused Rutgers upon presentation b/c they didn't understand how u can take a course multiple times in 1 term. Generally I think this language is okay-- for minority schools where it's possible to take it twice, we can address with them directly, but keep it simpler for the broader majority

Reply:
agreed that this is potentially confusing, it's one of the reasons i advocated for turning this set of features into boolean "has-taken" equivalents. i don't know if/how that ended up happening in a PR that kayla submitted a while back ...